src/e2e/java/teammates/e2e/cases/lnp/InstructorStudentEnrollmentLNPTest.java

```diff
@@ -232,6 +232,7 @@ public void runLnpTest() throws IOException {
     public void classTearDown() throws IOException {
         // There is no need to add the newly enrolled students to the JSON DataBundle#students. This is because the new
         // CourseStudent entities that were created are automatically deleted when the corresponding course is deleted.
+        generateResultAnalysis("InstructorStudentEnrollmentLNPTest", 0.2, 60000);
```

Comment:
Can you shift this to be the last line? Just due to some restrictions with the way the output is displayed in the build output